csrc/moe/topk_softmax_kernels.cu

```diff
@@ -25,11 +25,13 @@
     #include <cub/util_type.cuh>
     #include <cub/cub.cuh>
     #include <cuda/std/functional>
-    using AddOp = cuda::std::plus<float>;
+    using AddOp = cuda::std::plus<>;
+    using MaxOp = cuda::maximum<>;
```

Comment:
It seems like `maximum` does not currently work on 12.8 so we should guard on the CUDA version here